stress/shell/src/main/java/alluxio/stress/cli/MaxFileBench.java

```diff
@@ -0,0 +1,224 @@
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.AlluxioURI;
+import alluxio.annotation.SuppressFBWarnings;
+import alluxio.client.file.FileSystem;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.CreateFilePOptions;
+import alluxio.grpc.WritePType;
+import alluxio.stress.BaseParameters;
+import alluxio.stress.common.FileSystemClientType;
+import alluxio.stress.common.FileSystemParameters;
+import alluxio.stress.master.MasterBenchBaseParameters;
+import alluxio.stress.master.MasterBenchParameters;
+import alluxio.stress.master.MasterBenchTaskResult;
+import alluxio.stress.master.Operation;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.util.executor.ExecutorServiceFactories;
+import alluxio.util.io.PathUtils;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Strings;
+import org.apache.hadoop.fs.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * MaxFile StressBench class.
+ */
+@SuppressFBWarnings("BC_UNCONFIRMED_CAST")
+public class MaxFileBench extends StressMasterBench {
+  private static final Logger LOG = LoggerFactory.getLogger(MaxFileBench.class);
+
+  static AtomicBoolean sFinish = new AtomicBoolean(false);
+  private final MasterBenchTaskResult mTotalResults = new MasterBenchTaskResult();
+
+  private final List<String> mDefaultParams = Arrays.asList(
+      BaseParameters.BENCH_TIMEOUT, String.format("%ds", Integer.MAX_VALUE),
+      FileSystemParameters.CLIENT_TYPE, FileSystemClientType.ALLUXIO_NATIVE.toString(),
+      BaseParameters.CLUSTER_START_DELAY_FLAG, "0s",
+      MasterBenchParameters.DURATION, String.format("%ds", Integer.MAX_VALUE),
+      MasterBenchParameters.OPERATION, Operation.CREATE_FILE.toString(),
+      MasterBenchBaseParameters.STOP_COUNT, Integer.toString(
+          MasterBenchBaseParameters.STOP_COUNT_INVALID),
+      MasterBenchBaseParameters.WARMUP, "0s",
+      FileSystemParameters.WRITE_TYPE, WritePType.MUST_CACHE.toString()
+  );
+
+  /**
+   * @param args command-line arguments
+   */
+  public static void main(String[] args) {
+    // TODO(jenoudet): run MaxThroughput beforehand
+
+    mainInternal(args, new MaxFileBench());
+  }
+
+  @Override
+  public String getBenchDescription() {
+    List<String> descLines = new ArrayList<>(Arrays.asList(
+        "MaxFile. This stressbench runs CreateFile until the master becomes unresponsive.",
```

Comment:
let's not state the end condition but just say something like "creates file until no more files can be created."